apps/web/components/question/Question.css

```diff
@@ -17,6 +17,10 @@
   font: var(--typography-regular-large-body);
 }
 
+.web-QuestionTitle .ui-Button {
```

Comment:
for css naming conventions, do we want to standardize one of the following naming conventions? 
https://dev.to/ziizium/css-naming-conventions-5gd6

- BEM
- SMACSS
- ITCSS
- OOCSS
- AMCSS